FSQLocationBroker/FSQLocationBroker.m

```diff
@@ -516,13 +516,13 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
     BOOL isBackgrounded = applicationIsBackgrounded();
     CLLocation *mostRecentLocation = [locations firstObject];
     for (CLLocation *location in locations) {
-        if ([mostRecentLocation.timestamp laterDate:location.timestamp] == location.timestamp) {
+        if ([mostRecentLocation.timestamp compare:location.timestamp] == NSOrderedAscending) {
```

Comment:
This will always be false if `mostRecentLocation` is `nil` - I think it would be better to reverse it (i.e. `if ([location.timestamp compare:mostRecentLocation.timestamp] == NSOrderedDescending) {`) so that it will also pass if the incoming `location` is defined and the `mostRecentLocation` isn't.